app/presenters/answer_presenter.rb

```diff
@@ -1,4 +1,23 @@
 class AnswerPresenter < ContentItemPresenter
   include Body
   include LastUpdated
+
+  def ab_body
+    b_button = <<EOD
+      <div class="transaction">
+        <p class="get-started group">
+          <a id="epilepsy-driving-variant-b" class="button" href="https://www.driving-medical-condition.service.gov.uk/eligibility/entitlement-gb">
```

Comment:
The start page here:
https://www.gov.uk/report-driving-medical-condition

Links to:
https://www.driving-medical-condition.service.gov.uk/report